src/main/resources/native/android/c/launcher.c

```diff
@@ -224,9 +128,11 @@ static void *thread_func()
 {
     ssize_t rdsz;
     char buf[128];
-    while((rdsz = read(pfd[0], buf, sizeof buf - 1)) > 0) {
-        if(buf[rdsz - 1] == '\n') --rdsz;
-        buf[rdsz] = 0;  /* add null-terminator */
+    while ((rdsz = read(pfd[0], buf, sizeof buf - 1)) > 0)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Check buffer boundaries if used in a loop including recursive loops (CWE-120, CWE-20).  ](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=5120753)